includes/abstracts/calendar.php

```diff
@@ -570,11 +570,17 @@ public function set_time_format( $format = '' ) {
 	 */
 	public function set_datetime_separator( $separator = '' ) {
 
+		$separator_spacing = get_post_meta( $this->id, '_calendar_datetime_separator_spacing', true );
 		if ( empty( $separator ) ) {
 			$separator = get_post_meta( $this->id, '_calendar_datetime_separator', true );
 		}
 
-		$this->datetime_separator = esc_attr( $separator );
+		if ( empty( $separator_spacing ) ) {
+			$separator = '&nbsp;' . trim( $separator ) . '&nbsp;';
+		} else {
+			$separator = str_replace( ' ', '&nbsp;', $separator );
+		}
+		$this->datetime_separator = wp_kses_post( $separator );
```

Comment:
@zachwills The previous attribute escaping would strip out the spacing, so i used `wp_kses_post` so that wouldn't happen. I didn't really give thought to adding html here as this separator is not only output as html in the tooltip but added as data attributes for js, so I might need to be more specific on escaping here (i.e. didn't really test html here specifically, but I can do that). Thoughts?